nodejs/week1/homework/warmup.js

```diff
@@ -0,0 +1,22 @@
+console.log("inside warmup file");
+
+class Circle {
+    constructor(radius){
+        this.radius = radius
+    }
+    getRadius(){
```

Comment:
We already have the radius of the circle (this is a constructor of the class), so we should get the diameter instead. The calculation is good, but the name of the method needs to be fixed.